lib/build/web_components.dart

```diff
@@ -39,7 +39,7 @@ Asset generateWebComponentsBootstrap(Resolver resolver, Transform transform,
       document.head.querySelector('script[type="application/dart"]');
   for (var importPath in htmlImportRecorder.importPaths) {
     var import = new dom.Element.tag('link')
-      ..attributes = {'rel': 'import', 'href': importPath,};
+      ..attributes = (new Map<dynamic,String>()..addAll({'rel': 'import', 'href': importPath,}));
```

Comment:
This shouldn't be necessary either, and it should be a `Map<String, String>` afaik (see [here](https://api.dartlang.org/stable/1.18.1/dart-html/Element/attributes.html)). That type should be inferred though anyways, unless `importPath` isn't known to be a `String` or something?